--- a/Block/Dashboard/Graph.php	2019-03-21 10:13:02.771826000 +0200
+++ b/Block/Dashboard/Graph.php	2019-03-21 13:51:56.264336250 +0200
@@ -15,7 +15,7 @@
     /**
      * Api URL
      */
-    const API_URL = 'http://chart.apis.google.com/chart';
+    const API_URL = 'https://image-charts.com/chart';
 
     /**
      * All series
@@ -186,12 +186,12 @@
     public function getChartUrl($directUrl = true)
     {
         $params = [
-            'cht' => 'lc',
-            'chf' => 'bg,s,ffffff',
-            'chco' => 'ef672f',
-            'chls' => '7',
-            'chxs' => '0,676056,15,0,l,676056|1,676056,15,0,l,676056',
-            'chm' => 'h,f2ebde,0,0:1:.1,1,-1',
+            'cht'  => 'lc',
+            'chf'  => 'bg,s,f4f4f4|c,lg,90,ffffff,0.1,ededed,0',
+            'chm'  => 'B,f4d4b2,0,0,0',
+            'chco' => 'db4814',
+            'chxs' => '0,0,11|1,0,11',
+            'chma' => '15,15,15,15'
         ];
 
         $this->_allSeries = $this->getRowsData($this->_dataRows);
@@ -279,21 +279,12 @@
         $this->_axisLabels['x'] = $dates;
         $this->_allSeries = $datas;
 
-        //Google encoding values
-        if ($this->_encoding == "s") {
-            // simple encoding
-            $params['chd'] = "s:";
-            $dataDelimiter = "";
-            $dataSetdelimiter = ",";
-            $dataMissing = "_";
-        } else {
-            // extended encoding
-            $params['chd'] = "e:";
-            $dataDelimiter = "";
-            $dataSetdelimiter = ",";
-            $dataMissing = "__";
-        }
-
+        // Image-Charts Awesome data format values
+        $params['chd'] = "a:";
+        $dataDelimiter = ",";
+        $dataSetdelimiter = "|";
+        $dataMissing = "_";
+        
         // process each string in the array, and find the max length
         $localmaxvalue = [0];
         $localminvalue = [0];
@@ -329,44 +320,13 @@
 
         foreach ($this->getAllSeries() as $index => $serie) {
             $thisdataarray = $serie;
-            if ($this->_encoding == "s") {
-                // SIMPLE ENCODING
-                for ($j = 0; $j < sizeof($thisdataarray); $j++) {
-                    $currentvalue = $thisdataarray[$j];
-                    if (is_numeric($currentvalue)) {
-                        $ylocation = round(
-                            (strlen($this->_simpleEncoding) - 1) * ($yorigin + $currentvalue) / $yrange
-                        );
-                        $chartdata[] = substr($this->_simpleEncoding, $ylocation, 1) . $dataDelimiter;
-                    } else {
-                        $chartdata[] = $dataMissing . $dataDelimiter;
-                    }
-                }
-            } else {
-                // EXTENDED ENCODING
-                for ($j = 0; $j < sizeof($thisdataarray); $j++) {
-                    $currentvalue = $thisdataarray[$j];
-                    if (is_numeric($currentvalue)) {
-                        if ($yrange) {
-                            $ylocation = 4095 * ($yorigin + $currentvalue) / $yrange;
-                        } else {
-                            $ylocation = 0;
-                        }
-                        $firstchar = floor($ylocation / 64);
-                        $secondchar = $ylocation % 64;
-                        $mappedchar = substr(
-                            $this->_extendedEncoding,
-                            $firstchar,
-                            1
-                        ) . substr(
-                            $this->_extendedEncoding,
-                            $secondchar,
-                            1
-                        );
-                        $chartdata[] = $mappedchar . $dataDelimiter;
-                    } else {
-                        $chartdata[] = $dataMissing . $dataDelimiter;
-                    }
+            for ($j = 0; $j < sizeof($thisdataarray); $j++) {
+                $currentvalue = $thisdataarray[$j];
+                if (is_numeric($currentvalue)) {
+                    $ylocation = $yorigin + $currentvalue;
+                    array_push($chartdata, $ylocation . $dataDelimiter);
+                } else {
+                    array_push($chartdata, $dataMissing . $dataDelimiter);
                 }
             }
             $chartdata[] = $dataSetdelimiter;
